packages/eslint-config-angular/internal/extraneous-class.js

```diff
@@ -2,6 +2,7 @@ module.exports = {
   overrides: [
     {
       files: ['*.ts'],
+      plugins: ['@typescript-eslint'],
```

Comment:
sure, that not a parser, but plugins property?

Reply:
It may be worth specifying the parser in the future as well. By default it is pulled from base.js in tests.